src/test/java/org/utplsql/api/TestRunnerIT.java

```diff
@@ -25,7 +26,7 @@ public void runWithDefaultParameters() {
             Connection conn = db.newConnection();
             new TestRunner().run(conn);
         } catch (SQLException e) {
-            Assert.fail(e.getMessage());
+            fail(e);
```

Comment:
There is no need to re-fail just declare `throws SQLException` and remove `try`

Reply:
Thanks, will change that